### PR TITLE
Consider fedimint balance for on chain sends

### PR DESCRIPTION
**Pull request by TonyGiorgio**
Depends on https://github.com/MutinyWallet/mutiny-node/pull/1146

TODO:
- [x] we should say not to send more than once to the address
- [x] we should tell them it requires 10 confs for deposits

### CODE REVIEW

File: src/routes/Send.tsx [hunk 1]
```diff
@@ -825,6 +830,11 @@ export function Send() {
                             <p>{i18n.t("send.hodl_invoice_warning")}</p>
                         </InfoBox>
                     </Show>
+                    <Show when={source() === "onchain"}>
+                        <InfoBox accent="blue">
+                            {i18n.t("send.warn_address_reuse")}
+                        </InfoBox>
+                    </Show>
```
Comment (benthecarman) [inline src/routes/Send.tsx:837]: this needs to be on the receive side, not send side

Comment (futurepaul) [reply src/routes/Send.tsx]: okay thank you, fixed, moved to receive QR screen

File: src/routes/Receive.tsx [hunk 2]
```diff
@@ -459,6 +460,16 @@ export function Receive() {
                                 <p>{error()}</p>
                             </InfoBox>
                         </Show>
+                        <Show
+                            when={
+                                flavor() === "onchain" &&
+                                state.federations?.length
```
Comment (benthecarman) [inline src/routes/Receive.tsx:466]: we should just always show this for on-chain. we only sync an address once for our own on-chain wallet so we'll have a similar issue

Comment (futurepaul) [reply src/routes/Receive.tsx]: fixed

**Review by benthecarman — APPROVED** [review]

**Pull request merged**